### PR TITLE
Add PDF of transaction to ticket confirmation email

### PR DESCRIPTION
**Pull request by sbinlondon**
Fixes https://github.com/opencollective/opencollective/issues/3726

### CODE REVIEW

**Review by znarf — CHANGES_REQUESTED** [review]
In other similar places, we have the ability to disable the feature, we should have it here too.

Also, seems there is a legit problem with a GraphQL test.

**Comment by sbinlondon** [comment]
> In other similar places, we have the ability to disable the feature, we should have it here too.

We call `getTransactionPdf` here https://github.com/opencollective/opencollective-api/blob/f54f71aeb2de5b81585b3c913c09bd6bf6121103/server/lib/notifications.js#L125 and within `getTransactionPdf` we have the config check for `config.pdfService.fetchTransactionsReceipts`.

File: server/lib/notifications.js [hunk 1]
```diff
@@ -117,6 +120,19 @@ async function notifyUserId(UserId, activity, options = {}) {
     const parentCollective = await event.getParentCollective();
     const ics = await event.getICS();
     options.attachments = [{ filename: `${event.slug}.ics`, content: ics }];
+    const transaction = await models.Transaction.findOne({
+      where: { OrderId: activity.data.order.id, type: 'DEBIT' },
```
Comment (Betree) [inline server/lib/notifications.js:124]: Not sure if it has any impact, but in other places we take the credit transaction

```suggestion
      where: { OrderId: activity.data.order.id, type: 'CREDIT' },
```

**Review by Betree — COMMENTED** [review]
:ice_cream: Not a problem because tickets cannot have recurring contributions, but fetching the transaction by order ID can bring bad surprises with recurring contributions. If there's a transaction ID in `activity.data` it could be nice to use it.

File: server/lib/notifications.js (hunk 1 above)
Comment (sbinlondon) [reply server/lib/notifications.js:124]: I wondered that. The DEBIT amount looks weird but in my mind it made sense that the DEBIT was coming out of the user's account, vs the CREDIT being applied to the collective... will fix!

**Comment by sbinlondon** [comment]
> If there's a transaction ID in `activity.data` it could be nice to use it.

Not that I saw but I can look again

**Review by Betree — APPROVED** [review]
Haven't tested, but the code looks good to me

File: server/graphql/v2/mutation/AccountMutations.ts [hunk 2]
```diff
@@ -164,11 +164,11 @@ const accountMutations = {
   },
   removeTwoFactorAuthTokenFromIndividual: {
     type: new GraphQLNonNull(Individual),
-    description: 'Add 2FA to the Account if it does not have it',
+    description: 'Remove 2FA from the Account if it has been enabled',
```
Comment (znarf) [inline server/graphql/v2/mutation/AccountMutations.ts:167]: Not related to the PR.

**Review by znarf — APPROVED** [review]
Good for me!

**Pull request merged**